lib/projectPreprocessor.js

```diff
@@ -325,8 +326,8 @@ class ProjectPreprocessor {
 			throw new Error(`No specification version defined for extension ${extension.metadata.name}`);
 		} else if (extension.specVersion !== "0.1" && extension.specVersion !== "1.0") {
 			throw new Error(
-				`Invalid specification version defined for extension ` +
-				`${extension.metadata.name}: ${extension.specVersion}. ` +
+				`Invalid or unsupported specification version ${extension.specVersion} defined for extension ` +
```

Comment:
I would even put `unsupported` in front as this is probably the general case.
Or even just replace `Invalid` with `Unsupported` and don't mention `invalid` at all?

Reply:
Done